### ISSUE TITLE
Loadtafsir

### ISSUE THREAD

**Issue by hema995** [AlfaazPlus/QuranApp#389]
### Steps to reproduce

Idk

### Expected behavior

When i select tafsirs it should tafsirs appears 

### Actual behavior

It stays no tafsirs available also that happen with Reciter 

### QuranApp version

2023.09.03.1

### Android version

10

### Other details

java.net.UnknownHostException: Unable to resolve host "cdn.jsdelivr.net": No address associated with hostname
	at java.net.Inet6AddressImpl.lookupHostByName(Inet6AddressImpl.java:156)
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Inet6AddressImpl.java:103)
	at java.net.InetAddress.getAllByName(InetAddress.java:1152)
	at ea.p.b(SourceFile:218)
	at ha.e.a(SourceFile:131)
	at ha.e.b(SourceFile:1)
	at ga.a.a(SourceFile:441)
	at ia.f.b(SourceFile:143)
	at ia.f.a(Unknown Source:4)
	at ga.a.a(SourceFile:140)
	at ia.f.b(SourceFile:143)
	at ia.f.a(Unknown Source:4)
	at ia.a.a(SourceFile:247)
	at ia.f.b(SourceFile:143)
	at ia.a.a(SourceFile:506)
	at ia.f.b(SourceFile:143)
	at ia.f.a(Unknown Source:4)
	at o5.d.a(SourceFile:5)
	at ia.f.b(SourceFile:143)
	at ia.f.a(Unknown Source:4)
	at ea.z.d(SourceFile:87)
	at ea.y.a(SourceFile:19)
	at fa.a.run(SourceFile:18)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)
Caused by: android.system.GaiException: android_getaddrinfo failed: EAI_NODATA (No address associated with hostname)
	at libcore.io.Linux.android_getaddrinfo(Native Method)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:74)
	at libcore.io.BlockGuardOs.android_getaddrinfo(BlockGuardOs.java:200)
	at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:74)
	at java.net.Inet6AddressImpl.lookupHostByName(Inet6AddressImpl.java:135)
	... 25 more

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.

**Comment by faisalcodes** [comment]
Looks like jsdelivr is blocked by your service provider. May I know what is your service provider?

**Comment by Luciogi** [comment]
Try with VPN

**Comment by faisalcodes** [comment]
I will add a fallback proxy soon InshaAllah.
